scripts/build-packages-dist.sh

```diff
@@ -44,6 +44,10 @@ dirs=`echo "$targets" | sed -e 's/\/\/src\/\(.*\):npm_package/\1/'`
 # do this to ensure that the version placeholders are properly populated.
 for pkg in ${dirs}; do
   pkg_dir="${bazel_bin_path}/src/${pkg}/npm_package"
+  # Make all directories in the previous package output writable. Bazel by default
+  # makes tree artifacts and file outputs readonly. This causes permission errors
+  # when deleting the folder. To avoid these errors, we make all files writable.
+  chmod -R u+w ${pkg_dir}
```

Comment:
I know that this also makes files writable, but it would make things unnecessarily more platform dependent if we rely on `find` to filter out only directories. Nothing wrong with also making the files writable before deletion.

Technically files don't need to be writable for deletion since we have the `x` bit set in the folder permissions.